src/index.html

```diff
@@ -5,8 +5,429 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    >
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+      rel="stylesheet"
+    >
   </head>
-  <body>
-    <h1>Catalog</h1>
+  <body class="body">
+    <header class="header">
+      <a href="https://www.moyo.ua/ua/" class="header__logo">
+        <img src="images\logo.png" alt="logo">
+      </a>
+      <nav>
+        <li class="header__list">
+          <ul class="header__item">
```

Comment:
change tags. ul is a parent, li is a child
```suggestion
        <ul class="header__list">
          <li class="header__item">
```

Reply:
OMG. I wonder how is my code works 😅

Reply:
done